bin/texvcjs

```diff
@@ -45,10 +45,10 @@ PACKAGES.forEach(function(pkg) {
 
 // output result
 if (result.status === '+') {
-    util.print(result.status + (result.output || ''));
+    console.log(result.status + (result.output || ''));
```

Comment:
Can you undo this change. I think we should remove the emptyline or at least make this a configurable option. Especially when the output is consumed by other programs this might be a problem.